.github/workflows/ci.yml

```diff
@@ -36,14 +36,10 @@ jobs:
               yarn electron-builder --linux=deb
               sudo apt install -y --no-install-recommends ./dist/installers/*.deb
               ;;
-            macOS )
-              yarn electron-builder --mac=dir
-              cp -r dist/installers/mac/Brim.app /Applications
-              ;;
```

Comment:
Maybe leave this for the future and just change `if: runner.os != 'Windows'` to `if: runner.os == 'Linux'?

Reply:
Ok sounds good.